packages/sdk/src/chains/EVM/fee/basicFee.ts

```diff
@@ -41,7 +41,7 @@ export const calculateBasicfee = async ({
   tokenAmount: string;
   recipientAddress: string;
 }): Promise<FeeDataResult> => {
-  const depositData = constructDepositDataEvmSubstrate(tokenAmount, recipientAddress);
+  const depositData = createERCDepositData(tokenAmount, recipientAddress);
   // WHY 0X00 AND NOT 0X0?
```

Comment:
What's this? What do you want to do exactly?

Reply:
no, we can get rid of it